server/google_resources.py

```diff
@@ -33,12 +33,13 @@ def credentials_to_dict(credentials):
 
 def save_credentials(creds_dict):
   #TODO: Grab current user
-  current_user = UserModel.find_by_id("someone@gmail.com")
-  if current_user:
-    current_user.updateCredentials(creds_dict)
-    return "Credentials Saved."
-  else:
-    return "Current user not found."
+  # current_user = UserModel.find_by_id("someone@gmail.com")
```

Comment:
you can use the `@login_required` decorator which will also give you the current user (check out the /prospects endpoint for an example)

Reply:
Awesome, I added that - can you check it again and see if it looks like my understanding is correct?